pages/packages/maven.md

```diff
@@ -95,18 +94,54 @@ To download a package:
 
 ### Installing a package
 
+A Java package can be installed using code snippet details provided on the package's details page.
+
+To install a package:
+
 1. [Access the package's details](#access-a-packages-details).
 1. Ensure the _Installation_ > _Installation instructions_ section is displayed.
-1. Copy the code snippet and paste this into the `pom.xml` file (under the `project` XML tag) to install this package.
+1. Copy each code snippet, and paste them into their respective `~/.m2/settings.xml` and `pom.xml` files (under the `project` XML tag), and run `mvn install` on this modified `pom.xml` to install this package.
```

Comment:
Quick query... the following instruction:

> (under the `project` XML tag)

was extrapolated from code comments in the code snippet provided through the UI.

However, these code comments have since been removed. Therefore, is this instruction still worth keeping in the docs?

Reply:
The docs here should probably be more explicit, so leave them in :+1: